.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: C
+addons:
+    apt:
+        packages:
+        - libgmp-dev
+        - libmpfr-dev
+cache:
+    apt: true
+    #directories:
```

Comment:
Do you think we should still build the cross compiler, cache it, and build the kernel? It doesn't sound like a bad idea actually. If not, these lines need to go.